wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -8,46 +8,62 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
 + **Ensure that kudosu is awarded to all posts that deserve it.** Additionally consult the mapper before giving out kudosu to make sure there are not any reasons why the post does not need kudosu (e.g. the modder has received one already for a different post which you overlooked). If the modpost is too old to receive kudosu, consult the modder to make an additional post linking the original he receives kudosu for.
-+ **You must not place any icons besides the star, bubble, bubble pop and heart.** Whilst you are able to set other icons, they are either deprecated and discouraged from use (flame, circle, drum, apple, piano) or reserved for use by the Nomination Assessment Team (nuke). If you accidentally set an unwanted icon, contact a NAT immediately or overwrite the icon in case you wanted to post a bubble or heart.
+
+## Nomination
+
+### General
+
++ **Do not nominate beatmaps with unrankable issues.** The General Ranking Criteria and additionally all mode-specific Ranking Criteria apply to all sets with mode-specific difficulties.
++ **Do not nominate beatmapsets that you contributed to.** This applies to your own beatmaps, collaboration difficulties you participate in as well as beatmaps that you hitsounded or made a storyboard for.
++ **Do not nominate a beatmap directly without any communication with the mapper.** The mapper might have done self-changes after the last nomination, so nominating a beatmapset without communicating with the mapper can lead to pointless disqualifications.
 + **Do not nominate a beatmap without properly checking all difficulties of the beatmap all by yourself.** If you are unfamiliar with the game mode some of the difficulties are mapped in, ask fellow Beatmap Nominators for help. Improper checking includes just running a modding tool before nominating or not even downloading the beatmap.
++ **Do not qualify beatmaps of modes which you are not a full member of, unless they were nominated by someone who is.** This ensures that a full nominator is always part of the process for each mode.
++ **Hybrid beatmapsets must be nominated by full nominators from two of its modes in order to be qualified**, as well as gain clear approval from any remaining modes' nominators in the modding discussion. This means only full members may nominate hybrid sets, and ensures that a full nominator takes responsibility for each mode (regardless of nominating or just giving approval in the discussion). In this scenario, nominators proficient in multiple modes can approve more than one mode.
++ **Do not nominate beatmaps of game modes you are not formally qualified for.** You must be qualified to nominate at least one of the game modes included in the beatmapset by being a member of the respective part of the Beatmap Nominators in order to be allowed to nominate it.
+
+### Forum Nomination
```

Comment:
i think this whole section could be discarded. forum nominations aren't worth keeping a whole block of rules for when they are practically gone now (and if someone wants to nominate a map via that system, it's better to move it to v2 anyway since dqs are broken on v1)

Reply:
Discarded the section and moved the `Beatmaps qualified in modding v1 that need to be disqualified must be brought to the attention of the NAT` rule to disqualification so there's some guidance what to do in such a scenario.